src/less/components/steps.less

```diff
@@ -30,7 +29,6 @@
 
             &:hover {
                 background-color: @nearwhite;
-                box-shadow: 1px 0 0 0 @gray-bright, -1px 0 0 0 @gray-bright, 0 1px 0 0 @nearwhite;
```

Comment:
Is this intentional? This is a 1px line above and below a step item, a border-like presentation that does not effect the box model.

Reply:
I think i went through the component too fast and just removed anything related to box-shadow. I added the commented sections again.